hooks/useServicesList.ts

```diff
@@ -43,56 +41,23 @@ export const useServiceList = (listId: string): ServiceListHandler => {
       }
     }) ?? [] // cannot be undefined or useQueries throws an error
   const serviceQueryResults =
-    useQueries<UseQueryOptions<Service, Error>[]>(servicesQueryOptions)
+    useQueries<UseQueryOptions<Resource, Error>[]>(servicesQueryOptions)
   const isLoadingServices = serviceQueryResults.some(
     (result) => result.isLoading
   )
   const baseServices = serviceQueryResults.map((result) => result.data)
   const { filteredData: filteredServices, setSearchQuery } = useFilters(
-    isLoadingServices ? [] : (baseServices as Service[]),
-    ['name', 'description'],
-    'taxonomyString'
+    isLoadingServices ? [] : (baseServices as Resource[]),
+    ['title', 'details'],
+    'needs'
   )
 
-  const addressIdToServiceName: Record<string, string> = {}
-  const addressQueryOptions =
```

Comment:
Address fields are on resources in the new schema so we no longer need a separate query for them.